Tests/.flake8

```diff
@@ -41,11 +41,17 @@ ignore =
     # D400	First line should end with a period
     # D401	First line should be in imperative mood
     # D412	No blank lines allowed between a section header and its content
-    D400,D401
+    D400,D401,
     # We ignore D412 deliberately:
     D412,
     # =========================================
     # flake8-commas (in case installed locally)
     # =========================================
     # C812	missing trailing comma
-    C812
+    C812,
+    # ================================================
+    # flake8 whitespace: W50#
+    # ================================================
+    # W503	Line break before binary operator
+    # PEP8 is now in favour of line-breaks before binary operators
+    W503
```

Comment:
Can you put this at line 9 of the file with the other flake8 built in codes please?